sdk/rust/src/token/token_create_transaction.rs

```diff
@@ -312,6 +315,12 @@ impl TokenCreateTransaction {
 
 #[async_trait]
 impl TransactionExecute for TokenCreateTransactionData {
+    fn validate_checksums_for_ledger_id(&self, ledger_id: &LedgerId) -> Result<(), Error> {
+        // TODO: validate custom fee collector account IDs once that's merged
```

Comment:
custom fee collector account id requires a protobuf submodule update

(feel free to mark this note as resolved though)